cdc/processor.go

```diff
@@ -248,7 +177,7 @@ func NewProcessor(
 	}
 
 	for _, table := range p.status.TableInfos {
-		p.addTable(ctx, int64(table.ID), table.StartTs)
+		go p.addTable(ctx, int64(table.ID), table.StartTs)
```

Comment:
why using independent routines here

Reply:
A. we create a  processor follow the steps below:
1. create ddl puller
2. create schema builder
3. call `addTable` function
4. call `processor.Run` function, the `Run` of ddl puller will be called in this step. 

B. in `addTable` function
1. call `scheam builder` to create scheam storage, and `schema builder` will call `HandlePreviousDDLJobIfNeed` of new schema storage.
2. do something ...

In step `B.1`, if schema builder is unresolved,  `HandlePreviousDDLJobIfNeed` will return a `ErrUnresolved` error, and schema builder will retry and wait for it.
but ddl puller is still not running, so schema builder will never be resolved.

Reply:
Oh I see. 
`addTable` relies on `ddl puller`

Reply:
The processor startup procedure is not very clear, we can refine this later.